pkg/operation/botanist/component/extensions/operatingsystemconfig/original/components/kubelet/config_test.go

```diff
@@ -140,6 +141,7 @@ var _ = Describe("Config", func() {
 			ReadOnlyPort:              0,
 			ResolverConfig:            pointer.String("/etc/resolv.conf"),
 			RuntimeRequestTimeout:     metav1.Duration{Duration: 2 * time.Minute},
+			SeccompDefault:            pointer.Bool(false),
```

Comment:
Lets leave this a `nil` because we expect this to be defaulted only for >= 1.25 clusters.